xbmc/cores/VideoSettings.h

```diff
@@ -100,6 +100,8 @@ class CVideoSettings
   float m_VolumeAmplification;
   int m_SubtitleStream;
   float m_SubtitleDelay;
+  int m_subtitleVerticalPosition;
+  bool m_subtitleVerticalPositionSave;
```

Comment:
As long as we're adding new members, they should be initialized in the header:

```c++
  int m_subtitleVerticalPosition{0};
  bool m_subtitleVerticalPositionSave{false};
```

Reply:
done